packages/office-ui-fabric-react/src/components/Facepile/Facepile.scss

```diff
@@ -10,6 +10,11 @@ $spacingAroundItemButton: 2px;
   clear: both;
 }
 
+.itemContainer
+{
+  display:flex;
```

Comment:
why is this here? It doesn't seem to have any impact. Under  what circumstance is it used?